common/src/Model/BrushFace.h

```diff
@@ -154,6 +154,8 @@ namespace TrenchBroom {
             vm::vec3 center() const;
             vm::vec3 boundsCenter() const;
             FloatType area(vm::axis::type axis) const;
+            FloatType area() const;
+            bool onPlane(const vm::plane3d& plane) const;
```

Comment:
I feel the name "coplanarWith" would be a little more descriptive. It's not a big deal, so feel free to ignore.